tempest/api/workloadmgr/network/test_network_restore.py

```diff
@@ -342,77 +342,32 @@ def test_5_network_restore_cli_workload_reassign(self):
                     del vm_details_bf_2[vm_name]['tenant_id']
                     del vm_details_bf_2[vm_name]['security_groups']
 
-            klist = sorted([*nt_bf])

```

Comment:
Sorting of the networks list returned is not required ? Multiple times have observed that network details are same, but do not exactly match due to details returned in different order.

Reply:
Yes the script was failing due to this sorting code which was done only for network and subnet and not for instance and router. I tried execution multiple times and the results are consistence